cmd/dao_tests/pubkey_test.go

```diff
@@ -63,7 +75,120 @@ func TestSimple(t *testing.T) {
 	assert.Nil(t, nil)
 }
 
-func TestMain(t *testing.M) {
```

Comment:
Why is the TestMain gone? :(

Reply:
I could move it back but since there is no Setup or DatabaseCleanup outside the tests, it is not needed anymore. Or do you think it would be better to run it with TestMain?

Reply:
Well if it is in, it is clear entry point to the program. We can later on decide to structure the tests into multiple files and then it might be useful to keep the clear entry point, but you're right we do not need it at this point.

Tho I still belive that initialization (configuration and DB connection) should happen just once exactly in that function and not on every DB cleanup, but as long as it works :)